dockerfile/build.Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM gradle:7.2-jdk11-alpine AS build
+WORKDIR /home/gradle/src
+RUN git clone -b task_4 https://github.com/kkarnauk/se-project-fall-2021.git .
+RUN gradle build --no-daemon
```

Comment:
Why not use a wrapper? What happens if your project would update the Gradle version, but in Docker, you would not update it...?

Reply:
fix